src/client/src/apps/SpotlightRoute/InlineBlotter.tsx

```diff
@@ -21,11 +22,11 @@ const Table = styled.table`
 `
 
 interface IProps {
-  currency?: string
-  count?: number
+  readonly filters?: BlotterFilters
```

Comment:
`InlineBlotter` was not showing the same trades as main blotter (because it was only using currency, not symbol). we need to bring it in sync with main blotter. let's use already established api